src/test/store/profile-view.test.js

```diff
@@ -173,50 +176,67 @@ describe('actions/ProfileView', function() {
   });
 
   describe('selectTrack', function() {
-    describe('with a thread tracks', function() {
```

Comment:
(don't worry, this is just the setup got moved to the higher scope, and github is a bit crazy)